### PR TITLE
draft: proposal representative requirements

### PR DESCRIPTION
**Pull request by sendilkumarn**
Addresses #13 

r? @jorydotcom

### CODE REVIEW

**Comment by sendilkumarn** [comment]
That is a good point. But for that we should first address https://github.com/openjs-foundation/standards/issues/6 and then add budget information at the later stages if needed.

**Comment by sendilkumarn** [comment]
Thanks @MylesBorins 

This is exactly what I wanted to ask, thanks for the detailed description.  Opened #22 for adding Governance.

**Comment by jorydotcom** [comment]
I think this is a good start in terms of reqs - happy to keep thinking about it. I would like to see a 'reporting' requirement in that they should have to present, write, or otherwise report on the meeting to the community. I'd also assume they would need to adhere to the OpenJS Foundation's code of conduct but probably a good idea to make that explicit

File: proposals/stage0/REQUIREMENTS/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+The requirements for the elected representatives are as follows:
+
+* They should be a regular member at the OpenJS Foundation.
+* They should be available to travel for the meetings.
+* They should be ready to handle the conflicts of interest that might arise.
+* They should focus / address the concerns from the OpenJS Foundation and its projects' prespective.
+
+Questions that are essential to address before moving to next stage:
+* Will the existing communities are okay with OpenJS foundation nominating a representative?
+* How to ensure that these representatives do not become a backdoor for the the enterprises?
```
Comment (Eomm) [inline proposals/stage0/REQUIREMENTS/README.md]:
```suggestion
* How to ensure that these representatives do not become a backdoor for the enterprises?
```

File: proposals/stage0/REQUIREMENTS/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+The needs of projects and ecosystem stakeholders being delegated via the representatives. These representatives are vital for voicing the collective view of and for the foundation.
+
+## Requirements of the representatives
+
+The requirements for the elected representatives are as follows:
+
+* They should be a regular member at the OpenJS Foundation.
+* They should be available to travel for the meetings.
+* They should be ready to handle the conflicts of interest that might arise.
+* They should focus / address the concerns from the OpenJS Foundation and its projects' prespective.
```
Comment (mikesamuel) [inline proposals/stage0/REQUIREMENTS/README.md]: Perhaps another duty might be to highlight agenda items of interest to project stakeholders before meetings and provide summaries of conclusions reached during the meeting.

Comment (sendilkumarn) [reply proposals/stage0/REQUIREMENTS/README.md]: Good one let us add it. Thanks 👍

**Comment by sendilkumarn** [comment]
Thanks @jorydotcom for the comment. I updated responsibilities.

**Pull request merged**